github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml

```diff
@@ -1,45 +1,41 @@
 presubmits:
   kubevirt/terraform-provider-kubevirt:
-  - name: pull-terraform-provider-kubevirt-e2e-k8s-1.20
-    branches:
-      - main
-    annotations:
-      fork-per-release: "true"
-      testgrid-create-test-group: "false"
+  - name: pull-terraform-provider-kubevirt-e2e
     always_run: true
-    optional: false
-    skip_report: false
-    cluster: prow-workloads
+    optional: true
     decorate: true
+    skip_report: true
```

Comment:
if you want to see feedback on PRs - this needs to be set to false.

Reply:
Thanks, fixed